external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,560 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
          "label": "ice cream flavours"
       }
    }
+}
 ```
 
-In the example above there are two reused dimensions. For the first existing dimension, "reused column" takes the existing dimension "years" and reuses it without any changes. The second dimension is an example of the creation of a new dimension but showing that ice-cream flavours it is a child dimension of flavours.
+In the example above there are two reused dimensions. For the first existing dimension, "reused column" takes the existing dimension "years" and reuses it without any changes. The second dimension is an example of the creation of a new dimension but showing that ice-cream flavours is a child dimension of flavours.
 
 Unless the component being reused is a literal attribute and you're providing a `"data_type"` key-value pair, any other key-value pairs provided will change the column to a new component which will extend the linked parent component.
 
-## Shared column configuration options
+## Dimension Configuration
+
+The following fields can be configured for each dimension in your data set. csvcubed will assume that a column represents a dimension if the `type` field is left blank, or explicitly specified as `dimension`.
 
-There are several configuration options available across column types except observations.
+| **field name**      | **description**                                                                                                                                                                                                                                                                                 | **default value**                                                                               |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| `type`              | The type of the column (Required)                                                                                                                                                                                                                                                               | *dimension*                                                                                     |
```

Comment:
`The type of the column (Required)` we've just said it isn't required for dimensions.

Reply:
Think I'd like to split Required/Optional out into a separate column for all reference tables